deployment/grid/terraform/Makefile

```diff
@@ -21,15 +21,17 @@ init:
 # execute terraform with confirmation
 #########################################
 apply: $(TF_SRCS) $(GRID_CONFIG)
+	terraform apply -var-file $(GRID_CONFIG) -var="grafana_admin_password=$(GRAFANA_ADMIN_PASSWORD)" -target module.compute_plane.aws_iam_policy.eks_pull_through_cache_permission -target module.compute_plane.aws_iam_policy.agent_permissions
```

Comment:
Can't we use TF dependency trees to enforce this flow of applies?

Reply:
As much I have been able to test it, `terraform` was not able to compute the dependency tree correctly. The role is this target apply is to make sure `terraform` compute the dependency in the right order. 
I would suggest that we open a bug in order to track this in the future release of terraform or the aws provider.
What do you think ?